Dockerfile

```diff
@@ -5,7 +5,7 @@ ARG APP_INSIGHTS_AGENT_VERSION=2.5.1
 FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-1.2
 
 COPY lib/AI-Agent.xml /opt/app/
-COPY build/libs/spring-boot-template.jar /opt/app/
+COPY build/libs/bulk-scan-end-to-end-tests.jar /opt/app/
```

Comment:
do we need docker file and related config ?

Reply:
I don't think we need it. removed all related files